airflow/providers/google/cloud/triggers/gcs.py

```diff
@@ -97,3 +97,76 @@ async def _object_exists(self, hook: GCSAsyncHook, bucket_name: str, object_name
             if object_response:
                 return "success"
             return "pending"
+
+
+class GCSPrefixBlobTrigger(GCSBlobTrigger):
+    """
+    A trigger that fires and it looks for all the objects in the given bucket
+    which matches the given prefix if not found sleep for certain interval and checks again.
```

Comment:
```suggestion
    Looks for objects in bucket matching the prefix.  
    
    If none found, sleep for interval and check again.  Otherwise, return matches.
```